src/nodejs/jshttpclient.nim

```diff
@@ -1,5 +1,212 @@
-## HttpClient for JavaScript targets.
-## * Asyncronous `HttpClient` implemented on top of `jsfetch`.
-## * Syncronous `HttpClient` implemented on top of `jsxmlhttprequest`.
-import jsasynchttpclient, jssynchttpclient
-export jsasynchttpclient, jssynchttpclient
+## * HttpClient for JavaScript targets implemented as sugar on top of `fusion/js/jsxmlhttprequest <https://nim-lang.github.io/fusion/src/fusion/js/jsxmlhttprequest>` and `std/jsfetch <https://nim-lang.github.io/Nim/jsfetch>`_
+when not defined(js):
+  {.fatal: "Module jshttpclient is designed to be used with the JavaScript backend.".}
+
+import std/[asyncjs, jsheaders, jsfetch]
+import fusion/js/jsxmlhttprequest
```

Comment:
`jsxmlhttprequest` lives in this repo now, because basically Fusion is abandoned, 
here at least I can fix it, so change the import plz.